pyvista/utilities/reader.py

```diff
@@ -1392,20 +1392,9 @@ def add_q_files(self, files):
                 files = [files]
         files = [_process_filename(f) for f in files]
 
-        if hasattr(self.reader, 'AddFileName'):
-            # AddFileName was added to vtkMultiBlockPLOT3DReader sometime around
-            # VTK 8.2. This method supports reading multiple q files.
```

Comment:
Given the weakness of the language here, are we sure that it works in vtk versions 9.0+?  Is this in a test?

Reply:
It looks like there is a test for this, so it should be caught in the tests.